logbook/action.go

```diff
@@ -0,0 +1,22 @@
+package logbook
+
+import (
+	"github.com/qri-io/dataset"
+)
+
+// ActionType is the type of action that a logbook just completed
+type ActionType byte
+
+const (
+	// ActionMoveCursor is an action that moves the cursor for a dataset
+	ActionMoveCursor ActionType = iota
+)
+
+// Action represents the result of an action that logbook just completed
+type Action struct {
```

Comment:
This can stay here for now, but I think in the _very_ near future everything in this file should move to a new package: `event`. 

`Action` is a concrete type that currently bridges between `dscache` and `logbook`. To that end, I don't think this belongs in logbook, as it's not really logbook's primary concern